its/ruling/src/test/resources/expected/docker/docker-S2260.json

```diff
@@ -2,4 +2,16 @@
 'docker-project:ruling/src/test/resources/sources/docker/parsingError/Dockerfile':[
 1,
 ],
+'docker-project:sources/docker/docker-containers/alpine3.7_py3.5.5_analytics/Dockerfile':[
+18,
+],
+'docker-project:sources/docker/docker-containers/dockeransible/Dockerfile':[
+20,
+],
+'docker-project:sources/docker/docker-containers/pyalp/alpine3.7_py3.5.5/Dockerfile':[
+18,
+],
+'docker-project:sources/docker/docker-containers/pyalp/alpine3.8_py3.5.5/Dockerfile':[
+18,
+],
```

Comment:
I don't understand how those files could be parsed before ^^'
They all seems to have a parse error because they have trivias (comment) inside multi line instruction, which we don't support yet, and should not be related to this PR (or I missed something).

Reply:
Mmhh probably we had some spacing possible before in multiline instruction, which would take care of this.
So indeed for now we do not parse those files anymore, should be fixed by SONARIAC-533.